sig-architecture/charter.md

```diff
@@ -1,40 +1,175 @@
+
+    * Technical leads *MAY* decide to step down at anytime and propose a replacement, who must be approved by all of the other technical leads
+
+    * All technical leads cannot be from a single company
+
+* Subproject Owners
+
+    * Scoped to a subproject defined in[ sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1454)
+
+    * The initial owners *SHOULD* be established at subproject founding from relevant OWNERS files wherever possible
+
+    * *MUST* be an escalation point for technical discussions and decisions in the subproject
+
+    * *MUST* set milestone priorities or delegate this responsibility
+
+    * *MUST* remain active in the role and are automatically removed from the position if they are unresponsive for > 3 months and *MAY* be removed by consensus of other subproject owners and all of the Technical leads if not proactively working with other Subproject Owners to fulfill responsibilities.
+
+    * *MAY* decide to step down at anytime and propose a replacement. Use[ lazy-consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) amongst subproject owners with fallback on majority vote to accept proposal. This *SHOULD* be supported by a majority of subproject members (those having some role in the subproject).
+
+    * *MAY* select additional subproject owners through a[ super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote amongst subproject owners. This *SHOULD* be supported by a majority of subproject contributors (through[ lazy-consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) with fallback on voting).
+
+    * Number: at least 2
+
+    * Defined in[ sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1454)[ OWNERS](https://github.com/kubernetes/community/blob/master/committee-steering/governance/contributors/devel/owners.md) files
+
+* Members
+
+    * Currently is comprised of reviewers and approvers in[ OWNERS](https://github.com/kubernetes/community/blob/master/committee-steering/governance/contributors/devel/owners.md) files for subprojects
+
+## Organizational management
+
+* Six months after this charter is first ratified, it MUST be reviewed and re-approved by the SIG in order to evaluate the assumptions made in its initial drafting
+
+* SIG meets bi-weekly on zoom with agenda in meeting notes
+
+    * *SHOULD* be facilitated by chairs unless delegated to specific Members
+
+* SIG holds office hours bi-weekly on zoom with agenda in meeting notes
+
+    * *SHOULD* be facilitated by chairs unless delegated to specific Members
+
+* TODO: The SIG MUST make a best effort to provide leadership opportunities to individuals who represent different races, national origins, ethnicities, genders, abilities, sexual preferences, ages, backgrounds, levels of educational achievement, and socioeconomic statuses
```

Comment:
No objection to this, but it seems like it belongs in the overall project governance charter (sttering committee) than in a specific sig charter.